keyboards/linworks/fave87h/config.h

```diff
@@ -18,7 +18,7 @@ along with this program. If not, see <http://www.gnu.org/licenses/>.
 #pragma once
 
 /* Define RGB */
-#define RGB_MATRIX_LED_COUNT 87
+#define RGB_MATRIX_LED_COUNT 127
```

Comment:
```suggestion
#define RGB_MATRIX_LED_COUNT 87
```
Conflict on this file - this define has been removed as the CLI can determine the LED count from the `g_led_config`.